roles/addons/lvm/vars/RedHat.yml

```diff
@@ -0,0 +1,2 @@
+lvm_packages_to_install:
+  - lvm2
```

Comment:
OpenSUSE also provides the lvm2 package. There is no need of OS specific var file is there is no OS specific value.
Can you move this parameter to vars/main.yml?

Reply:
Done